docs/extend/index.md

```diff
@@ -75,13 +77,16 @@ enabled, and use it to create a volume.
 
 3.  Create a volume using the plugin.
     This example mounts the `/remote` directory on host `1.2.3.4` into a
-    volume named `sshvolume`. This volume can now be mounted into containers.
+    volume named `sshvolume`.   
+   
+    This volume can now be mounted into containers.
 
     ```bash
     $ docker volume create \
       -d vieux/sshfs \
       --name sshvolume \
-      -o sshcmd=user@1.2.3.4:/remote
+      -o sshcmd=user@1.2.3.4:/remote \
+      -o password=password_for_user_on_remote_host
```

Comment:
Can do something like `$(cat file_containing_password)` or something, so at least then it wouldn't be in the bash history as well?